### DISCUSSION TITLE
Canal Panda (Spain)

### DISCUSSION THREAD

**Discussion by ELProductions** [iptv-org/community#791] [Q&A]
### Channel Name

Canal Panda (Spain)

### Channel ID

CanalPanda.es

### Website

canalpanda.es

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
According to our database this channel was closed on 15 December 2022.

https://iptv-org.github.io/channels/es/CanalPanda
